dotcom-rendering/src/components/Discussion/Comments.tsx

```diff
@@ -136,6 +136,7 @@ export const Comments = ({
 	const [isCommentFormActive, setIsCommentFormActive] = useState<boolean>(
 		!!commentBeingRepliedTo,
 	);
+	const [userNameMissing, setUserNameMissing] = useState<boolean>(false);
```

Comment:
This can be inferred if you like:

```suggestion
	const [userNameMissing, setUserNameMissing] = useState(false);
```

Reply:
> We're quite divided

Then just pick the one you prefer!